docs/tutorials/gettingstartedexample/gse-tap-basic.md

```diff
@@ -15,20 +15,28 @@ This tutorial will teach you the steps needed to configure a tap acquisition usi
 
 ## Prerequisites
 
-Review and complete all setup from the [PXIe-1486 Getting Started Guide](https://www.ni.com/docs/en-US/bundle/pxie-1486-getting-started/) or [PXIe-1487 Getting Started Guide](https://www.ni.com/docs/en-US/bundle/pxie-1487-getting-started/).
+Review and complete all setup from the appropriate getting started guide:
+- [PXIe-1486 Getting Started Guide](https://www.ni.com/docs/en-US/bundle/pxie-1486-getting-started/)
+- [PXIe-1487 Getting Started Guide](https://www.ni.com/docs/en-US/bundle/pxie-1487-getting-started/)
+- [PXIe-1488 Getting Started Guide](https://www.ni.com/docs/en-US/bundle/pxie-1488-getting-started/)
+- [PXIe-1489 Getting Started Guide](https://www.ni.com/docs/en-US/bundle/pxie-1489-getting-started/)
 
 Basic knowledge of LabVIEW and LabVIEW FPGA concepts. 
 
 A camera supported by the getting started example configuration scripts (i.e. Leopard Imaging IMX490).
 
-A supported interface module that includes both serial input and serial output channels (tap module) and a second supported interface module that includes serial input channels (acq module) on a PXI system running Windows.
+A supported interface module that includes both serial input and serial output channels (tap module) and a second supported interface module that includes serial input channels (acq module) on a PXI system running Windows. The table below lists all supported tap and acq modules and supported cameras that have configuration scripts provided with the getting started example. Configuration scripts for cameras not listed in the table below need to be provided separately.
```

Comment:
Suggest rewording for clarity at a minimum, but see the further comment below for a more comprehensive potential change:

"For cameras not listed in the table below, configuration scripts need to be provided separately."

Reply:
I made the suggested change.